src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/invalid/AUTOTEST1.json

```diff
@@ -0,0 +1 @@
+AC-3
```

Comment:
this file should be removed. it was also introduced accidentally under a wrong folder.